src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -804,8 +806,9 @@ class Volumes(list):
     to filter them via keyword arguments.
     """
 
-    def __init__(self):
-        self._populate()
+    def __init__(self, empty=False):
+        if not empty:
```

Comment:
I understand the need for the variable and then the check, but it reads backwards: "if not empty then populate". If something is not empty it means it has contents (or at least that is how I am reading it). You could maybe create a variable from the action? In this case the method is `_populate` so you could rename it to populate:

```suggestion
def __init__(self, populate=True):
        if populate:
            self._populate()
```

Reply:
Absolutely agreed. Made the change. Thanks!